notebooks/sam_tests/cmip6_density_transform_pipeline_dev.ipynb

```diff
@@ -8,7 +8,7 @@
     {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    foldername = "density_remapping_time_yearly_final"
</pre><p>We can probably find a bette name? I would suggest <code>density_remapped_annual</code>,  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jbusecke/cmip6_omz/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>